numpy/core/tests/test_umath.py

```diff
@@ -4160,6 +4160,10 @@ def test_against_cmath(self):
                 b = cfunc(p)
                 assert_(abs(a - b) < atol, "%s %s: %s; cmath: %s" % (fname, p, a, b))
 
+    @pytest.mark.xfail(
+        _glibc_older_than("2.17"),
```

Comment:
It is 2.17, so "older than" won't xfail the test, right?

Reply:
right, I was copying the pattern from elsewhere. I will update it to 2.18